packages/server/src/fhir/operations/find.test.ts

```diff
@@ -405,4 +424,192 @@ describe('Schedule/:id/$find', () => {
+            status: 'free',
+            schedule: createReference(schedule),
+            serviceType: [{ coding: [{ code: 'new-patient', system: 'http://example.com' }] }],
+          },
+        },
+        {
+          resource: {
+            resourceType: 'Slot',
+            start: formatNY(new Date('2025-12-04T13:00:00.000-05:00')),
+            end: formatNY(new Date('2025-12-04T13:30:00.000-05:00')),
+            status: 'free',
+            schedule: createReference(schedule),
+            serviceType: [{ coding: [{ code: 'new-patient', system: 'http://example.com' }] }],
+          },
+        },
+      ],
+    });
+  });
+
+  test('when no matching serviceType is found, uses any wildcard availability', async () => {
+    const schedule = await makeSchedule({
+      'new-patient': { availability: fridayOnly, duration: 45, bufferBefore: 15 },
+      'office-visit': { availability: twoDaySchedule, duration: 30, alignmentOffset: 15, alignmentInterval: 30 },
+      [wildcard]: { availability: fourDayWorkWeek, duration: 20, alignmentOffset: 30 },
+    });
+    const response = await request
+      .get(`/fhir/R4/Schedule/${schedule.id}/$find`)
+      .set('Authorization', `Bearer ${project.accessToken}`)
+      .set('Content-Type', ContentType.FHIR_JSON)
+      .query({
+        start: formatNY(new Date('2025-12-04T10:00:00.000-05:00')),
+        end: formatNY(new Date('2025-12-04T14:00:00.000-05:00')),
+        'service-type': 'http://example.com|new-patient-visit',
+      });
+    expect(response.body).not.toHaveProperty('issue');
+    expect(response.status).toBe(200);
+    expect(response.body).toMatchObject<Bundle>({
+      resourceType: 'Bundle',
+      type: 'searchset',
+      entry: [
+        // wildcard slots are 20 minutes long, start on the half-hour, and have no `serviceType` attribute
+        {
+          resource: {
+            resourceType: 'Slot',
+            start: formatNY(new Date('2025-12-04T10:30:00.000-05:00')),
+            end: formatNY(new Date('2025-12-04T10:50:00.000-05:00')),
+            status: 'free',
+            schedule: createReference(schedule),
+          },
+        },
+        {
+          resource: {
+            resourceType: 'Slot',
+            start: formatNY(new Date('2025-12-04T11:30:00.000-05:00')),
+            end: formatNY(new Date('2025-12-04T11:50:00.000-05:00')),
+            status: 'free',
+            schedule: createReference(schedule),
+          },
+        },
+        // No entry at 12:30; there is a lunch gap in the scheduled availability
+        {
+          resource: {
+            resourceType: 'Slot',
+            start: formatNY(new Date('2025-12-04T13:30:00.000-05:00')),
+            end: formatNY(new Date('2025-12-04T13:50:00.000-05:00')),
+            status: 'free',
+            schedule: createReference(schedule),
+          },
+        },
+      ],
+    });
+  });
+
+  test('returns appropriate slots for each serviceType passed', async () => {
```

Comment:
@rahul1 Does this seem like the appropriate behavior? The question that came up was what to do if multiple service-type options are passed that have differing availability requirements. My answer was to surface all the options as their parameter groups specify, but it means that your results can have different codes and durations (and alignment/buffer/...) amongst all the results. You also can have overlapping slots in the result set if they came from different scheduling parameter groups.